internal/quota/client_test.go

```diff
@@ -14,43 +14,51 @@ import (
 )
 
 func TestGetQuota_Success(t *testing.T) {
-	// given
-	response := Response{
-		Plan:  "aws",
-		Quota: 2,
+	tests := []struct {
+		name     string
+		response map[string]any
+		expected int
+	}{
+		{
```

Comment:
Since we get float and use int to round the value, please add test case with real va